setup-pg-user.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+usr=manageusr
```

Comment:
Shouldn't you be using environment variables? or pass in the values?
script file name says `setup-pg-user.sh` shouldnt it be more generic to match up with the readme otherwise i would expect the script to setup pg user